tests/phpunit/unit/api/actions/beansUnsetAction.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Tests for _beans_unset_action()
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
+ *
+ * @since   1.5.0
+ */
+
+namespace Beans\Framework\Tests\Unit\API\Actions;
+
+use Beans\Framework\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_BeansUnsetAction
+ *
+ * @package Beans\Framework\Tests\Unit\API\Actions
+ * @group   unit-tests
+ * @group   api
+ */
+class Tests_BeansUnsetAction extends Test_Case {
+
+	/**
+	 * Available action statuses.
+	 *
+	 * @var array
+	 */
+	protected $action_status;
+
+	/**
+	 * Action to be/is registered.
+	 *
+	 * @var array
+	 */
+	protected $action;
+
+	/**
+	 * The json_encode() version of the above action.
+	 *
+	 * @var string
+	 */
+	protected $encoded_action;
+
+	/**
+	 * Setup test fixture.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		$this->action_status = array( 'added', 'modified', 'removed', 'replaced' );
+		require_once BEANS_TESTS_LIB_DIR . 'api/actions/functions.php';
+		require_once BEANS_TESTS_LIB_DIR . 'api/utilities/functions.php';
+
+		$this->action         = array(
+			'hook'     => 'foo',
+			'callback' => 'callback',
+			'priority' => 10,
+			'args'     => 1,
+		);
+		$this->encoded_action = wp_json_encode( $this->action );
+	}
+
+	/**
+	 * Reset the test fixture.
+	 */
+	protected function tearDown() {
+		parent::tearDown();
+
+		global $_beans_registered_actions;
+		$_beans_registered_actions = array(
+			'added'    => array(),
+			'modified' => array(),
+			'removed'  => array(),
+			'replaced' => array(),
+		);
+	}
+
+	/**
+	 * Test _beans_unset_action() should return false when action's configuration is not registered.
+	 */
+	public function test_should_set_and_return_action() {
```

Comment:
comment and function name are not in alignment.
maybe `public function test_should_return_false_when_not_registered` ?